### PR TITLE
wthrr 1.2.0 (new formula)

### PR DESCRIPTION
**Pull request by ivaquero**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

**Comment by chenrui333** [comment]
there is no need to keep creating new prs.

**Review by chenrui333 — APPROVED** [review]

**Comment by ivaquero** [comment]
> there is no need to keep creating new prs. (you can do the commit squash and forced push to refresh the PR)

I will focus on the same one next time. Thank you for all your guidance.

File: Formula/w/wthrr.rb [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  def install
+    system "cargo", "install", *std_cargo_args
+  end
+
+  test do
+    assert_match version.to_s, shell_output("#{bin}/wthrr --version")
+    system bin/"wthrr", "-h"
+
+    require "pty"
+    require "io/console"
```
Comment (carlocab) [inline Formula/w/wthrr.rb:25]:
```suggestion
```
Don't think you use this anywhere?

**Review by carlocab — APPROVED** [review]
One small suggestion, otherwise LGTM. Thanks for your work here!

**Comment by carlocab** [comment]
One last code suggestion here: https://github.com/Homebrew/homebrew-core/pull/191593#discussion_r1771512049

File: Formula/w/wthrr.rb (hunk 1 above)
Comment (ivaquero) [reply Formula/w/wthrr.rb:25]: Let me check.

**Comment by ivaquero** [comment]
My git script makes my git always pulls before pushes and sometimes sets `HEAD` to the wrong branch, which causes a lot of mess. Very sorry for wasting the public resources. 

I solved the bug, and this will not happen again.

**Review by carlocab — APPROVED** [review]
No problem; thank you for your nice work!

**Pull request merged**